ecl/dedicated_hypervisor/v1/vcenter.py

```diff
@@ -68,3 +71,19 @@ def update(self, session, vcenter_id, password, license_id):
         )
         self._translate_response(resp, has_body=True)
         return self
+
+
+class Link_local_addresses(VCenter):
+    base_path = '/vcenters/addresses'
+
+    # capabilities
+    allow_list = True
+    allow_get = False
+    allow_create = False
+    allow_delete = False
+
+    # Properties
+    #: IP address.
+    link_local_address = resource2.Body('link_local_address')
```

Comment:
```suggestion
    address = resource2.Body('address')
```
API仕様書を確認いただくとわかりますが `link_local_addres` ではなく `address` です。

Reply:
ご指摘の箇所について修正します。